src/diff.ts

```diff
@@ -121,6 +127,17 @@ export async function getDiff(
     throw new Error('Schemas not defined');
   }
 
+  if (options.sort) {
+    [leftSchema, rightSchema] = [
```

Comment:
If you use the `lexicographicSortSchema` function of `graphql` this should be more simple:

```ts
[leftSchema, rightSchema] = [
  lexicographicSortSchema(leftSchema),
  lexicographicSortSchema(rightSchema)
]
```